tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/types/intersection/IntersectionTypeTest.java

```diff
@@ -238,11 +243,10 @@ public void testUnsupportedIntersectionNegative() {
                 BCompileUtil.compile("test-src/types/intersection/unsupported_intersection_negative.bal");
         int index = 0;
         validateError(result, index++, "unsupported intersection", 17, 8);
-        validateError(result, index++, "unsupported intersection", 18, 9);
         validateError(result, index++, "unsupported intersection", 19, 9);
         validateError(result, index++, "unsupported intersection", 21, 1);
-        validateError(result, index++, "unknown type 'A'", 23, 14);
-        validateError(result, index++, "unknown type 'II'", 23, 19);
+//        validateError(result, index++, "unknown type 'A'", 23, 14);
```

Comment:
Why these are commented? if not used, we can remove it, otherwise link related issue.

Reply:
This has to removed

Reply:
Removed [dbb563c](https://github.com/ballerina-platform/ballerina-lang/pull/39589/commits/dbb563c5fd05a206f691be51f3a3fdb07b082e18)